posthog/queries/query_date_range.py

```diff
@@ -14,9 +14,28 @@
 from posthog.utils import DEFAULT_DATE_FROM_DAYS
 
 
-# Assume that any date being sent from the client is timezone aware according to the timezone that the team has set
 class QueryDateRange:
```

Comment:
Rather than extend this already complicated base class to do even more - could we use inheritance/centralize the time-line specific functionality to timeline?

Reply:
We could, but the two new options are useful in a pretty abstract way, so I'm not convinced a new properties timeline-specific class would be clearer.